libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/MapboxNavigator.java

```diff
@@ -78,4 +78,11 @@ FixLocation buildFixLocationFromLocation(Location location) {
       provider
     );
   }
+
+  private Float checkNo(Float value) {
```

Comment:
What about this updated naming?

```suggestion
  private Float checkFor(Float value) {
```

or 

```suggestion
  private Float checkNoValueFor(Float value) {
```